tests/meson.build

```diff
@@ -324,17 +324,87 @@ if enable_pytest
     'test_usb.py',
     'test_wallpaper.py',
   ]
+
+  template_files = [
+    'templates/access.py',
+    'templates/account.py',
+    'templates/appchooser.py',
+    'templates/background.py',
+    'templates/clipboard.py',
+    'templates/email.py',
+    'templates/filechooser.py',
+    'templates/geoclue2.py',
+    'templates/globalshortcuts.py',
+    'templates/inhibit.py',
+    'templates/__init__.py',
+    'templates/inputcapture.py',
+    'templates/lockdown.py',
+    'templates/notification.py',
+    'templates/print.py',
+    'templates/remotedesktop.py',
+    'templates/screenshot.py',
+    'templates/settings.py',
+    'templates/usb.py',
+    'templates/wallpaper.py',
+  ]
+
   foreach pytest_file : pytest_files
     testname = pytest_file.replace('.py', '')
     test(
       'pytest/@0@'.format(testname),
       pytest,
-      args: pytest_args + ['-k', testname],
+      args: [meson.current_source_dir()] + pytest_args + ['-k', testname],
       env: pytest_env,
       suite: ['pytest'],
       timeout: 120,
     )
   endforeach
+
+  if enable_installed_tests
+    install_data(
+      pytest_files,
+      '__init__.py',
+      'conftest.py',
+      'asan.suppression',
+      install_dir: installed_tests_dir / 'tests',
+    )
+    install_data(
+      template_files,
+      install_dir: installed_tests_dir / 'tests' / 'templates',
+    )
+
+    installed_env = {
+      'XDG_DESKTOP_PORTAL_PATH': libexecdir / 'xdg-desktop-portal',
+      'XDG_PERMISSION_STORE_PATH': libexecdir / 'xdg-permission-store',
+      'XDG_DOCUMENT_PORTAL_PATH': libexecdir / 'xdg-document-portal',
+      'XDP_VALIDATE_ICON': libexecdir / 'xdg-desktop-portal-validate-icon',
+      'XDP_VALIDATE_SOUND': libexecdir / 'xdg-desktop-portal-validate-sound',
+    }
```

Comment:
Do we need to force all of these? Can't we just leave some (ideally all) of them at their defaults, which verifies that they will work as intended on end-user systems with no special environment variable overrides?

Reply:
I expect that we will need to force anything that is an input to the pytest scripts, but we shouldn't need to force anything that is only run indirectly (via the production executables) and gets a suitable default hard-coded into the executables.

So we probably do need to force `*_PATH`, but not `XDP_VALIDATE_ICON` or `XDP_VALIDATE_SOUND`.